kartothek/io/dask/repartition.py

```diff
@@ -0,0 +1,94 @@
+import math
+from typing import Optional
+
+import dask
+import dask.dataframe as dd
+
+import kartothek.io.dask.dataframe as ktk_ddf
+from kartothek.core.factory import DatasetFactory
+from kartothek.serialization import PredicatesType
+
+
+def optimize(ddf: dd.DataFrame) -> dd.DataFrame:
+    """
+    Optimize a dask graph using the dask.dataframe optimizer.
+    Parameters
+    ----------
+    ddf
+        Dask Dataframe
+    Returns
+    --------
+    dask.DataFrame
+    """
+    return dask.optimize(ddf, optimizations=[dask.dataframe.optimize])[0]
+
+
+def calculate_repartition_ratio(
+    dataset_factory: DatasetFactory,
+    table_name: str,
```

Comment:
I suggest to drop this. We don't want to support multi tabled datasets anymore. Therefore, I suggest to not offer the possibility to select a table for any new function.

Reply:
Done. Please check and confirm.